config/routes.rb

```diff
@@ -78,6 +80,11 @@
     resources :status_updates, :except => [:new]
   end
 
+  namespace :supervisor do
+    get 'dashboard', to: 'dashboard#index'
+  end
+
+
   # get 'activities(.:format)', to: 'activities#index', as: :activities
```

Comment:
Being picky here, but the extra newlines aren't needed, just one empty newline is enough.